src/components/AvatarWrapper/AvatarWrapper.vue

```diff
@@ -184,26 +182,38 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@import '../../assets/avatar';
-
 .avatar-wrapper {
-	height: 44px;
-	width: 44px;
-	border-radius: 44px;
-	@include avatar-mixin(44px);
-
-	&--small {
-		height: 22px;
-		width: 22px;
-		border-radius: 22px;
-		@include avatar-mixin(22px);
-	}
+	height: var(--avatar-size);
+	width: var(--avatar-size);
+	border-radius: var(--avatar-size);
+
+	.avatar {
+		position: sticky;
+		top: 0;
+		width: var(--avatar-size);
+		height: var(--avatar-size);
+		line-height: var(--avatar-size);
+		font-size: calc(var(--avatar-size) / 2);
+		border-radius: 50%;
+
+		&.icon {
+			background-color: var(--color-background-darker);
+		}
+
+		&.bot {
+			padding-left: 5px;
+			background-color: var(--color-background-darker);
+		}
 
-	&--medium {
-		height: 32px;
-		width: 32px;
-		border-radius: 32px;
-		@include avatar-mixin(32px);
+		&.guest {
+			color: #ffffff;
+			background-color: #b9b9b9;
```

Comment:
I would suggest removing these static colors. 
This is the view from Guest ( display name : pop ) 
![image](https://github.com/nextcloud/spreed/assets/84044328/9089499e-945d-4757-a943-24231870ebf4)
And this is from a logged in user and Guest is present in the call ( display name : pop) 
![image](https://github.com/nextcloud/spreed/assets/84044328/25654bee-e73d-429a-9a27-ef05f28519b9)

It would be nice if we use the same style for guest avatar from the user account view in the guest view ( make it always uppercase too)

Reply:
These colors are also used in other places, to identify guest person in participants list or messages, for example:
![image](https://github.com/nextcloud/spreed/assets/93392545/0e18bf23-93f5-4064-ba28-8fb974e451e9)

Although I can't reproduce your second screenshot, Maybe, it's related to issue in second comment

Reply:
This identification method might be okay in other places like messages and participant lists because it doesn't own much of "visual weight" but for call view, it occupies more visual surface. 
Alternative : Add "( guest )" to the display name to identify the guest.

Reply:
Please reference all this in follow-up issue =)